code/lib/preview-api/src/modules/store/csf/prepareStory.ts

```diff
@@ -50,19 +50,38 @@ export function prepareStory<TRenderer extends Renderer>(
   // anything at render time. The assumption is that as we don't load all the stories at once, this
   // will have a limited cost. If this proves misguided, we can refactor it.
 
-  const { moduleExport, id, name } = storyAnnotations;
+  return prepareAnnotations(storyAnnotations, componentAnnotations, projectAnnotations);
+}
+
+export function prepareMeta<TRenderer extends Renderer>(
+  componentAnnotations: NormalizedComponentAnnotations<TRenderer>,
+  projectAnnotations: NormalizedProjectAnnotations<TRenderer>
+): PreparedStory<TRenderer> {
```

Comment:
```suggestion
): PreparedMeta<TRenderer> {
```

Let's make a proper `PreparedMeta` return type rather than this in-between business.